xwiki-platform-distribution/xwiki-platform-distribution-flavor/xwiki-platform-distribution-flavor-test/xwiki-platform-distribution-flavor-test-webstandards/src/test/it/org/xwiki/test/webstandards/framework/AbstractValidationTest.java

```diff
@@ -217,6 +218,11 @@ protected static void addXarFiles(Class< ? extends AbstractValidationTest> valid
         }
     }
 
+    private static boolean isNotTechnicalPage(XarEntry entry)
+    {
+        return DefaultXarEntryType.HINT.equals(entry.getEntryType());
```

Comment:
This is a very different logic. Default type is used to protect pages which are not supposed to be modified by users which is very different from pages user are not supposed to see.

Reply:
OK. I wasn't quite sure about that :)